### PR TITLE
Updated my scripts to the current versions in my own repos

### PR DESCRIPTION
**Pull request by pstray**

### CODE REVIEW

File: scripts/uptime.pl [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
 
 use strict;
 use Irssi;
 use Irssi::Irc;
 use Irssi::TextUI;
 
-# ======[ Script Header ]===============================================
-
 use vars qw{$VERSION %IRSSI};
 ($VERSION) = '$Revision: 1.6 $' =~ / (\d+\.\d+) /;
```
Comment (vague666) [inline scripts/uptime.pl:11]: up version?

File: scripts/upgradeinfo.pl [hunk 2]
```diff
@@ -1,50 +1,34 @@
...
 #
 
 use strict;
 use Irssi 20021204.1123;
 use Irssi::TextUI;
 
-# ======[ Script Header ]===============================================
-
 use vars qw{$VERSION %IRSSI};
 ($VERSION) = '$Revision: 1.7 $' =~ / (\d+\.\d+) /;
```
Comment (vague666) [inline scripts/upgradeinfo.pl:10]: Here too

File: scripts/keepnick.pl [hunk 3]
```diff
@@ -1,38 +1,27 @@
...
 #
 
 use strict;
 use Irssi 20011118.1727;
 use Irssi::Irc;
 
-# ======[ Script Header ]===============================================
-
 use vars qw{$VERSION %IRSSI};
 ($VERSION) = '$Revision: 1.19 $' =~ / (\d+\.\d+) /;
```
Comment (vague666) [inline scripts/keepnick.pl:10]: version here too, @ailin-nemui maybe has an opinion if version needs to be updated when only cosmetic changes

File: scripts/dccmove.pl [hunk 4]
```diff
@@ -1,22 +1,19 @@
...
-#    $Id: dccmove.pl,v 1.4 2007/04/17 21:32:30 peder Exp $
-#
-# Copyright (C) 2003 by Peder Stray <peder@ninja.no>
+# Copyright (C) 2003-2021 by Peder Stray <peder.stray@gmail.com>
 #
 
 use strict;
 use Irssi;
 use vars qw{$VERSION %IRSSI};
 ($VERSION) = '$Revision: 1.4 $' =~ / (\d+\.\d+) /;
```
Comment (vague666) [inline scripts/dccmove.pl:8]: ++

File: _data/scripts.yaml [hunk 5]
```diff
@@ -5138,13 +5138,13 @@
...
+  contact: peder.stray@gmail.com
   description: 'Try a little harder to figure out client uptime'
   filename: uptime.pl
   license: GPL
-  modified: '2014-10-24 23:01:30'
+  modified: '2021-08-20 13:47:51'
   name: uptime
-  url: http://ninja.no/irssi/uptime.pl
+  url: https://github.com/pstray/irssi-uptime
   version: '1.6'
```
Comment (vague666) [inline _data/scripts.yaml:5148]: ++

File: _data/scripts.yaml [hunk 6]
```diff
@@ -5116,13 +5116,13 @@
...
+  contact: peder.stray@gmail.com
+  description: 'Statusbar item notifying you about updated binary'
   filename: upgradeinfo.pl
   license: GPL
-  modified: '2008-05-17 17:39:09'
+  modified: '2021-08-20 13:47:51'
   name: upgradeinfo
-  url: http://ninja.no/irssi/upgradeinfo.pl
+  url: https://github.com/pstray/irssi-upgradeinfo
   version: '1.7'
```
Comment (vague666) [inline _data/scripts.yaml:5126]: ++

File: _data/scripts.yaml [hunk 7]
```diff
@@ -2817,13 +2817,13 @@
...
+  contact: peder.stray@gmail.com
+  description: 'Convert various color codes to ANSI colors, useful for log filtering and viewing.'
   filename: log2ansi.pl
   license: GPL
-  modified: '2010-02-13 13:59:47'
+  modified: '2021-08-20 13:53:04'
   name: log2ansi
-  url: http://ninja.no/irssi/log2ansi
+  url: https://github.com/pstray/irssi-log2ansi
   version: '1.9'
```
Comment (vague666) [inline _data/scripts.yaml:2827]: ++

File: _data/scripts.yaml [hunk 8]
```diff
@@ -2599,13 +2599,13 @@
...
+  contact: peder.stray@gmail.com
   description: 'Try to get your nick back when it becomes available.'
   filename: keepnick.pl
   license: GPL
-  modified: '2019-05-04 20:18:06'
+  modified: '2021-08-20 13:47:51'
   name: keepnick
-  url: http://ninja.no/irssi/keepnick.pl
+  url: https://github.com/pstray/irssi-keepnick
   version: '1.19'
```
Comment (vague666) [inline _data/scripts.yaml:2609]: ++

File: _data/scripts.yaml [hunk 9]
```diff
@@ -1840,13 +1840,13 @@
...
+  contact: peder.stray@gmail.com
+  description: 'Basically an autoop script with a nice interface and nick coloring ;)'
   filename: friends_peder.pl
   license: GPL
-  modified: '2014-10-17 22:54:13'
+  modified: '2021-08-20 13:47:51'
   name: friends
-  url: http://ninja.no/irssi/friends.pl
+  url: https://github.com/pstray/irssi-friends
   version: '1.34'
```
Comment (vague666) [inline _data/scripts.yaml:1850]: ++

File: _data/scripts.yaml [hunk 10]
```diff
@@ -1322,13 +1322,13 @@
...
+  contact: peder.stray@gmail.com
   description: 'Move completed dcc gets to the subfolder done'
   filename: dccmove.pl
   license: GPL
-  modified: '2014-10-17 20:34:00'
+  modified: '2021-08-20 13:47:51'
   name: dccmove
-  url: http://ninja.no/irssi/dccmove.pl
+  url: https://github.com/pstray/irssi-dccmove
   version: '1.4'
```
Comment (vague666) [inline _data/scripts.yaml:1332]: ++

File: scripts/friends_peder.pl [hunk 11]
```diff
@@ -1,32 +1,22 @@
...
 use Irssi::Irc;
 use Irssi::TextUI;
 
-use Data::Dumper;
-$Data::Dumper::Indent = 1;
-
-# ======[ Script Header ]===============================================
-
 use vars qw{$VERSION %IRSSI};
 ($VERSION) = '$Revision: 1.34 $' =~ / (\d+\.\d+) /;
```
Comment (vague666) [inline scripts/friends_peder.pl:11]: ++

**Review by vague666 — COMMENTED** [review]
log2ansi version was not updated in script.yaml, although shouldn't script.yaml be updated automatically? For the other version numbers in scripts, Nei might have an opinion

**Comment by pstray** [comment]
Updated all the scripts with three-tuple semantic versions, so that shoud fix new versions in all of them, even though there were not all that had functionality updates :)

**Review by vague666 — APPROVED** [review]
Looks good to my eyes, thanks Peder :)

**Comment by pstray** [comment]
not sure why log2ansi still is on 1.9 in the scripts.yaml though... it should have been updated to 1.11.1

**Comment by pstray** [comment]
and why did upgradeinfo loose its 'command: upgradeinfo' in scripts.yaml?

**Comment by vague666** [comment]
Maybe @bw1 or @ailin-nemui has an idea, can't remember who added the scripts.yaml automation

**Comment by pstray** [comment]
seems upgradeinfo is stuck on the old version-number too...

File: scripts/upgradeinfo.pl [hunk 12]
```diff
@@ -1,50 +1,34 @@
...
 
 use strict;
 use Irssi 20021204.1123;
 use Irssi::TextUI;
 
-# ======[ Script Header ]===============================================
-
 use vars qw{$VERSION %IRSSI};
-($VERSION) = '$Revision: 1.7 $' =~ / (\d+\.\d+) /;
+($VERSION) = '$Revision: 1.7.1 $' =~ / (\d+(\.\d+) /;
```
Comment (vague666) [inline scripts/upgradeinfo.pl:10]: +) missing in regex

**Comment by pstray** [comment]
That fixed the missing 'command: upgradeinfo' too it seems...

**Comment by pstray** [comment]
log2ansi still seems to not get updated in in scripts.yaml though...

**Comment by pstray** [comment]
That fixed scripts.yaml for log2ansi too

**Pull request merged**